rust/sealevel/client/src/cmd_utils.rs

```diff
@@ -149,6 +149,8 @@ fn attempt_program_deploy(
         buffer_keypair_path.to_str().unwrap(),
         "--verbose",
         "--use-rpc",
+        "--max-len",
+        "500000",
```

Comment:
This was nice for testing when I was adding logs that increased the Solana program size.

Reply:
But I can remove it if others believe it won't be useful in the future.